base_station/gui/src/components/LCMEcho.vue

```diff
@@ -178,7 +179,8 @@
           {'topic': '/microscope', 'type': 'Microscope'},
           {'topic': '/zed_gimbal_cmd', 'type': 'ZedGimbalPosition'},
           {'topic': '/zed_gimbal_data', 'type': 'ZedGimbalPosition'},
-          {'topic': '/zero_position', 'type': 'ZeroPosition'}
+          {'topic': '/zero_position', 'type': 'ZeroPosition'},
+          {'topic': '/pdb_data', 'type': 'PDBData'}
```

Comment:
just a comment it looks like this was attempted to be alphabetical in the past can you maintain that for readability of this code base